Gemfile

```diff
@@ -3,22 +3,22 @@ gem 'rake'
 gem 'middleman', '3.3.9' # Bug when starting server in 3.3.10
 
 # Bourbon + Neat
-gem 'slim', '~> 2.1'
-gem 'sass', '3.4.5' # 3.4.6 has a regression
-gem 'bourbon', '~> 4.0'
-gem 'neat', '~> 1.7'
-gem 'bitters', '~> 0.10'
+gem 'slim', '~> 3.0.3'
+gem 'sass', '~> 3.4.13'
+gem 'bourbon', '~> 4.2.2'
+gem 'neat', '~> 1.7.2'
+gem 'bitters', '~> 1.0.0'
 
 # Livereload
-gem 'middleman-livereload', '~> 3.1.0'
+gem 'middleman-livereload', '~> 3.4.2'
 
 # Crush dems images
-gem 'middleman-imageoptim', '~> 0.1'
-gem 'image_optim', '~> 0.9'
+gem 'middleman-imageoptim', '~> 0.2.1'
+gem 'image_optim', '~> 0.20.2'
 
 # Deployment Solution
-gem 'middleman-deploy', '~> 0.3'
+gem 'middleman-deploy', '~> 1.0.0'
 
 # Testing
-gem 'capybara', '~> 2.2'
-gem 'rspec', '~> 3.0'
+gem 'capybara', '~> 2.4.4'
+gem 'rspec', '~> 3.2.0'
```

Comment:
@ajrobertson I don't think we want to version lock this pessimistically. Can you revise to keep the same number of decimal places (usually 2) for each gem? Let me know if you have any questions about that.